internal/pkg/cli/validate.go

```diff
@@ -61,7 +59,6 @@ var (
 
 	// Aurora-Serverless-specific errors.
 	errInvalidRDSNameCharacters    = errors.New("value must start with a letter")
-	errRDWSNotConnectedToVPC       = fmt.Errorf("%s requires a VPC connection", manifest.RequestDrivenWebServiceType)
```

Comment:
What do you think about retaining all the changes in `validate.go` this way it fails fast telling the users that RDWS cannot be associated with Aurora as it requires VPC connectivity?